backend/main/chapters/c06_lists.py

```diff
@@ -105,6 +105,61 @@ class double_numbers(ExerciseStep):
 
     This - is - a - list
 
+        """
+        hints = """
+This is similar to the previous exercise. You can start with your solution from that.
+You need to modify the string that you are adding at each step.
+In addition to each word in the list, you also have to add the separator.
+But you don't want to add the separator after adding the last word in the list.
+Unfortunately there is no "subtraction" with strings; you can't add the last separator then remove it.
+This is tricky! Using a for loop, how can you perform an operation at all steps except the last? Well, you can't (for now).
+What if you reversed it? Add the separator first, add the word second?
+This way, you have to add the separator at each step except the FIRST step.
+The solution is somewhat similar to the program from the "Understanding Programs with Snoop" section.
+This can be done with a for loop (with an if statement inside) and a boolean variable (to keep track of whether we are at the first step or not).
+Define a boolean variable BEFORE the for loop. Then change its value INSIDE the loop (after adding the necessary strings).
+        """
+
+        # TODO: MessageStep: catch the "obvious solution" where the user adds the separator after the last word?
```

Comment:
It sounds like you're saying they submitted a solution to the previous exercise. I don't see why we would specially catch that. But maybe the description of the task should clarify the difference further.

Reply:
Ah no... I meant the solution
```python
for word in words:
    total += word + separator
```
which would result in `This-is-a-list-`. Maybe this would not be a common mistake as I thought?

Reply:
Oh right, sorry, I got what you said but I'd forgotten what the previous exercise was and was thinking of an older exercise. Yes, a message for this is a good idea.